Elements/test/ObstacleTests.cs

```diff
@@ -235,7 +235,7 @@ public void IntersectsObstacleFromLine()
             Assert.True(horizontalObstacle.Intersects(horizontalLineOnBottom));
 
             var horizontalLineOnSide = horizontalLine.TransformedLine(new Transform(offset, 0, 0));
-            Assert.False(horizontalObstacle.Intersects(horizontalLineOnSide));
```

Comment:
@wynged @DmytroMuravskyi This image shows this test geometrically. The `horizontalLine` is in the plane of the face of the obstacle. This is rightly classified as an intersection by our code now, but was previously asserted to be false. Why?

![image](https://user-images.githubusercontent.com/1139788/202291731-3c2c31db-dd3d-4b94-8323-d6a1835d6375.png)